dev-python/nuitka/nuitka-0.6.2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_5 python3_6)
+inherit distutils-r1
+
+DESCRIPTION="Python to native compiler"
+HOMEPAGE="http://www.nuitka.net"
+SRC_URI="http://nuitka.net/releases/${P^}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```

Comment:
(not saying this is wrong, but distutils should put setuptools already in DEPEND or BDEPEND, depending on EAPI used.)

Reply:
probably isn't wrong, but being explicit is good.